File: js/javascript-practice/basics/js/task2.js
let range1;
let range2;
range1 = Number(prompt("Please enter the first number:"));
range2 = Number(prompt("Please enter the second number:"));

for (let i = range1; i <= range2; i++) {
    let isPrime = true;

    if (i <= 1) {
        isPrime = false;
    } else {
        for (let j = 2; j < i; j++) {
            if (i % j === 0) {
                isPrime = false;
                break;
            }
        }
    }

    if (isPrime) {
        console.log(i); 
    }
}

File: c/ctask/task2.c
#include<stdio.h>
int main()
{
    int num[10];
    printf("please enter the number: ");
    for(int i=1;i<10;i++)
    {
        scanf("%d",&num[i]);
            

    }
    printf("*************\n");
    for(int k=1;k<10;k++)
    {
      printf("user enter number: %d\n",num[k]);
    }
    
    

}